TriggertrapUIKit/HorizontalPicker.m

```diff
@@ -177,14 +188,17 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
         for (UIView *subview in [cell.contentView subviews]) {
             [subview removeFromSuperview];
         }
-        
-        UILabel *textLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, [self width] / 3, [self height])];
-        
+        UILabel *textLabel;
+        if (@available(iOS 11.0, *)) {
+            textLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, [self width] / 3, [self height])];
```

Comment:
Hey @alex-taffe, Im not sure if this is necessary. Both cases in the if/else look the same to me.

Reply:
My bad, was playing around with stuff and forgot to remove that once it became identical